gemini/tuning/vertexai_supervised_tuning_token_count_and_cost_estimation.ipynb

```diff
@@ -0,0 +1,819 @@
+{
```

Comment:
<p>Are you intended to keep output in the notebook. It's possible but wanted to make sure.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/867/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Only for a few cells, because it provides users with an indication of what they can expect from the output.